plugins/upload-assets-to-s3/README.md

```diff
@@ -21,13 +21,13 @@ plugins:
 ## Options
 | Key | Description | Default value |
 |-|-|-|
-| `accessKeyIdEnvVar` | variable name of the project's aws access key id | no default value - for backwards compatability the plugin falls back to the default value for `accessKeyId` |
+| `accessKeyIdEnvVar` | variable name of the project's aws access key id. If uploading to multiple buckets the same credentials will need to work for both. | no default value - for backwards compatability the plugin falls back to the default value for `accessKeyId` |
 | `secretAccessKeyEnvVar` | variable name of the project's aws secret access key | no default value - for backwards compatability the plugin falls back to the default value for `secretAccessKey` |
 | `accessKeyId` | **DEPRECATED** variable name of the project's aws access key id | 'aws_access_hashed_assets' |
 | `secretAccessKey` | **DEPRECATED** variable name of the project's aws secret access key | 'aws_secret_hashed_assets' |
 | `directory` | the folder in the project whose contents will be uploaded to S3 | 'public' |
-| `reviewBucket` | the development or test S3 bucket | ['ft-next-hashed-assets-preview'] |
-| `prodBucket` | production S3 bucket/s; an array of strings. The same files will be uploaded to each | ['ft-next-hashed-assets-prod', 'ft-next-hashed-assets-prod-us'] |
+| `reviewBucket` | the development or test S3 bucket | `['ft-next-hashed-assets-preview']` |
+| `prodBucket` | production S3 bucket/s; an array of strings. The same files will be uploaded to each. Note: most Customer Products buckets that have a `prod` and `prod-us` version are already configured in AWS to replicate file changes from one to the other so you don't need to specify both here. Also, if multiple buckets are specified the same credentials will need to be valid for both for the upload to be successful | `['ft-next-hashed-assets-prod']` |
```

Comment:
**suggestion(non-blocking)**: maybe we could put the note in italics so that it's easier for people to differentiate it when they're skimming the table?

```suggestion
| `prodBucket` | production S3 bucket/s; an array of strings. The same files will be uploaded to each. _Note: most Customer Products buckets that have a `prod` and `prod-us` version are already configured in AWS to replicate file changes from one to the other so you don't need to specify both here. Also, if multiple buckets are specified the same credentials will need to be valid for both for the upload to be successful_ | `['ft-next-hashed-assets-prod']` |
```